packages/codefixes/src/codefixInformationMap.generated.ts

```diff
@@ -0,0 +1,461 @@
+// <auto-generated />
+// generated from 'packages/codefixes/src/codefixesMessages.json'
+
+import { Diagnostics, DiagnosticMessage } from './diagnosticInformationMap.generated.js';
+
+export interface CodefixMessage {
+  name: string;
+  title: string;
+  description: string;
+  passing: string[];
+  failing: string[];
+  diagnostics: DiagnosticMessage[];
+}
+
+function fix(
+  name: string,
+  title: string,
+  description: string,
+  diagnostics: DiagnosticMessage[],
+  passing: string[],
+  failing: string[]
+): CodefixMessage {
+  return { name, title, description, passing, failing, diagnostics };
+}
+
+/** @internal */
+export const CodeFixes = {
+  addMissingAsync: fix(
+    'addMissingAsync',
+    'Add Missing `async` Keyword',
```

Comment:
Why do we need these messages to "copy" to our repo if we always have access to then when they are available to apply?